openapi/data_repository_service.swagger.yaml

```diff
@@ -164,6 +164,8 @@ definitions:
         type: string
         description: |-
           A string that can be used to name an `Object`.
+          This string is made up of uppercase and lowercase letters, decimal digits, hypen, period, and underscore [A-Za-z0-9.-_]. See https://tools.ietf.org/html/rfc3986#section-2.3[RFC 3986 § 2.3].
+          All RFC 3986 § 3986 reserved characters including whitespace in DRS IDs must be url-encoded [!*'();:@&=+$,/?#[] ]. See https://tools.ietf.org/html/rfc3986#section-2.2[RFC 3986 § 2.2].
```

Comment:
I suggest deleting the sentence on URL-encoding  -- these aren't IDs, and they don't get passed in URLs so don't need to be encoded.

Reply:
I changed this to say name, the reference to ID was a typo.